Utils/azuretools-core/src/com/microsoft/azuretools/sdkmanage/AzureManagerBase.java

```diff
@@ -249,14 +247,33 @@ protected AzureTokenCredentials getCredentials(String tenantId) {
     }
 
     private List<Tenant> getTenants(Azure.Authenticated authentication) {
-        return authentication.tenants().listAsync()
-                .onErrorResumeNext(err -> {
-                    LOGGER.warning(err.getMessage());
-                    return Observable.empty();
-                })
+        ResponseWrapper<List<Tenant>> responseWrapper = new ResponseWrapper<>();
```

Comment:
seems no need to create  a `ResponseWrapper` instance in this method?
it only needs tenants(data), responseWrapper is useless in this method.